src/op_analytics/transforms/fees/update/04_agg_daily_trace_to.sql

```diff
@@ -0,0 +1,257 @@
+
+  , SUM(preagg_sum_tx_l2_fee_native_minus_subtraces) AS sum_tx_l2_fee_native_minus_subtraces
+  , sumIf(preagg_sum_tx_l2_fee_native_minus_subtraces, tx_success) AS sum_tx_l2_fee_native_minus_subtraces_tx_success
+
+  , SUM(preagg_sum_tx_l2_base_fee_native_minus_subtraces) AS sum_tx_l2_base_fee_native_minus_subtraces
+  , SUM(preagg_sum_tx_l2_priority_fee_native_minus_subtraces) AS sum_tx_l2_priority_fee_native_minus_subtraces
+  , SUM(preagg_sum_tx_l2_legacy_base_fee_native_minus_subtraces) AS sum_tx_l2_legacy_base_fee_native_minus_subtraces
+
+  , SUM(preagg_sum_tx_fee_native_amortized_by_call) AS sum_tx_fee_native_amortized_by_call
+  , sumIf(preagg_sum_tx_fee_native_amortized_by_call, tx_success) AS sum_tx_fee_native_amortized_by_call_tx_success
+
+  , SUM(preagg_sum_tx_l2_fee_native_amortized_by_call) AS sum_tx_l2_fee_native_amortized_by_call
+  , sumIf(preagg_sum_tx_l2_fee_native_amortized_by_call, tx_success) AS sum_tx_l2_fee_native_amortized_by_call_tx_success
+
+  , SUM(preagg_sum_tx_l1_fee_native_amortized_by_call) AS sum_tx_l1_fee_native_amortized_by_call
+  , sumIf(preagg_sum_tx_l1_fee_native_amortized_by_call, tx_success) AS sum_tx_l1_fee_native_amortized_by_call_tx_success
+
+  , SUM(preagg_sum_tx_l2_base_fee_native_amortized_by_call) AS sum_tx_l2_base_fee_native_amortized_by_call
+  , SUM(preagg_sum_tx_l2_priority_fee_native_amortized_by_call) AS sum_tx_l2_priority_fee_native_amortized_by_call
+
+  , SUM(preagg_sum_tx_fee_native_l1_amortized_l2_minus_subtraces) AS sum_tx_fee_native_l1_amortized_l2_minus_subtraces
+
+  -- Calls
+  , SUM(preagg_count_top_level_calls) AS count_top_level_calls
+  --count non-null trace addresses, null is the transaction-level call.
+  , SUM(preagg_count_internal_calls_all_types) AS count_internal_calls_all_types
+  , SUM(preagg_count_internal_calls_all_types_trace_success) AS count_internal_calls_all_types_trace_success
+  --static calls only read state, and can not write
+  , SUM(preagg_count_internal_calls_static_type) AS count_internal_calls_static_type
+  , SUM(preagg_count_internal_calls_static_type_trace_success) AS count_internal_calls_static_type_trace_success
+  --delegate calls call a function on another contract
+  , SUM(preagg_count_internal_calls_delegate_type) AS count_internal_calls_delegate_type
+  , SUM(preagg_count_internal_calls_delegate_type_trace_success) AS count_internal_calls_delegate_type_trace_success
+  --normal function calls
+  , SUM(preagg_count_internal_calls_call_type) AS count_internal_calls_call_type
+  , SUM(preagg_count_internal_calls_call_type_trace_success) AS count_internal_calls_call_type_trace_success
+
+  -- Experimental
+  , countIf(is_transaction_with_internal_call_type_call) AS count_transactions_called_with_internal_type_call
+  , countIf(is_transaction_with_internal_call_type_call_or_delegate) AS count_transactions_called_with_internal_type_call_or_delegate
+
```

Comment:
We can cut the `tx_sucess` variants here:

`sum_trace_gas_used_minus_subtraces_tx_success_called_with_internal_type_call` through `sum_tx_fee_native_amortized_by_call_tx_success_called_with_internal_type_call`